### PR TITLE
Add a font setting option

### PR DESCRIPTION
**Pull request by Xzonn**
Use different fonts by default based on the language, and allow users to choose the font they want to use. It seems to run normally on Windows in Simplified Chinese, Japanese and English.

<details>
<summary>Screenshots</summary>

English
Simplified Chinese
Japanese
</details>

### CODE REVIEW

File: src/SerialLoops/ViewModels/Dialogs/PreferencesDialogViewModel.cs [hunk 1]
```diff
@@ -106,6 +108,19 @@ public void Initialize(PreferencesDialog preferencesDialog, Config config, ILogg
                             RequireRestart = true;
                         },
                     },
+                    new ComboBoxOption([
+                            ("", string.Format(Strings.Default_Font_Display, Strings.Default_Font)),
+                            ..SystemFonts.Collection.Families.Select(_ => (_.Name, _.Name)).ToList(),
```
Comment (jonko0493) [inline src/SerialLoops/ViewModels/Dialogs/PreferencesDialogViewModel.cs:113]:
```suggestion
                            ..SystemFonts.Collection.Families.Select(_ => (_.Name, _.Name)),
```
collection expression here means we can use the `IEnumerable` without converting to a `List`, which optimizes slightly

**Review by jonko0493 — APPROVED** [review]
With the one nit, this looks great. Especially clever thinking with loading the default font through the localized strings!! Great work all around.

**Review by WiIIiam278 — APPROVED** [review]
Approving to unblock bar jonko's nit, nice stuff!

**Comment by jonko0493** [comment]
Thinking about this more, this is also cool because it will let people pick, like, dyslexia-friendly fonts and other accessibility-related things! Very neat.

**Pull request merged**